actions/access_keys/search.go

```diff
@@ -17,21 +17,18 @@ import (
 // @Description	Search access key
 // @Tags		Access-key
 // @Produce		json
-// @Param		page query int false "page"
-// @Param		page_size query int false "page_size"
-// @Param		order_by_field query string false "order_by_field"
-// @Param		sort_direction query string false "sort_direction"
-// @Param		metadata query string false "metadata"
-// @Param		conditions query string false "conditions"
-// @Success		200
+// @Param		SearchRequestParameters body actions.SearchRequestParameters false "Supports targeted resource searches with filters for metadata and custom conditions, plus options for pagination and sorting to streamline data exploration and analysis"
```

Comment:
When keys are not found I got 500 error ? Is it desired behaviour ?

Reply:
We are returning 500 error every time something went wrong in engine method. Maybe we should create another task to check all method and return empty array instead of error if no object were found? I do think that it should not be and error here but I think it should be changed in methods not here